admin/src/scenes/volontaires/view/details.js

```diff
@@ -259,9 +224,21 @@ export default function VolontaireViewDetails({ young }) {
               ) : null}
               <Bloc title="Consentements">
                 <Details title={`Consentements validés par ${young.firstName} ${young.lastName}`} value={t(young.consentment || "false")} style={{ border: "none" }} />
-                <ExpandComponent>{youngConsentmentText}</ExpandComponent>
+                <ExpandComponent>
+                  <ul>
+                    {CONSENTMENT_TEXTS?.young?.map((line) => (
+                      <li key={line}>• {line}</li>
+                    ))}
+                  </ul>
+                </ExpandComponent>
                 <Details title="Consentements validés par ses représentants légaux" value={t(young.parentConsentment || "false")} style={{ border: "none" }} />
-                <ExpandComponent>{parentsConsentmentText}</ExpandComponent>
+                <ExpandComponent>
+                  <ul>
+                    {CONSENTMENT_TEXTS?.parents?.map((line) => (
+                      <li key={line}>• {line}</li>
```

Comment:
il y a 2 `•`je propose d'enlever celui qu'on ajoute et de laisser celui que génère le `li` ?
```suggestion
                      <li key={line}>{line}</li>
```
<img width="345" alt="Screenshot 2022-01-07 at 13 50 52" src="https://user-images.githubusercontent.com/33058720/148546469-b738a816-52e0-4a39-a80b-01c04f901f73.png">

Reply:
c'est dans la partie details du volontaire , coté `admin`